src/components/floatingContainer/floatingContainer.scss

```diff
@@ -0,0 +1,11 @@
+:host {
+  display: inline-block;
+}
+
+.floating-btn-wrapper {
+  display: flex;
+  position: fixed;
+  right: 16px;
+  bottom: 24px;
```

Comment:
Maybe double-check these position values. Our page gutter is 16px on mobile, 32px larger screens. There is a variable `--kd-page-gutter` too.

Reply:
Addressed.